modeltree.h

```diff
@@ -36,38 +36,50 @@
 namespace Asn1Acn {
 namespace Internal {
 
+class ProjectContentHandler;
+
 class ModelTree : public QObject
 {
     Q_OBJECT
 
     ModelTree();
     ~ModelTree() = default;
 
+    friend class ProjectContentHandler;
```

Comment:
TBH it is not. Right now it really bothers me that the way to safely operate on data storage and model tree is via usage of newly introduced class - ProjectContentHandler. This is because prior to changing model or storage signals need to be emitted. So i tried to hide modifiers of tree model behind friend class, but I though over it again, and would like to back away from idea of using friend class.